packages/nimble-components/src/anchor/template.ts

```diff
@@ -0,0 +1,47 @@
+import { html, ref, slotted, ViewTemplate } from '@microsoft/fast-element';
+import type {
+    AnchorOptions,
+    FoundationElementTemplate
+} from '@microsoft/fast-foundation';
+import type { Anchor } from '.';
+
+export const template: FoundationElementTemplate<
+ViewTemplate<Anchor>,
+AnchorOptions
```

Comment:
FastAnchor and AnchorOptions include the start and end properties. We should include them in the template and hide them in styles or make sure our types are cleaned correctly.

Reply:
I had to inline the `${startSlotTemplate(context, definition)}` and `${endSlotTemplate(context, definition)}` because they contain leading and trailing whitespace that causes the trailing underlined space issue. I could have also just put them both before the content slot, but that seems worse.